src/Widgets/SaveDialog.vala

```diff
@@ -195,6 +195,7 @@ public class Screenshot.SaveDialog : Granite.Dialog {
         content.add (format_cmb);
         content.add (location_label);
         content.add (location_button);
+        content.show_all ();
```

Comment:
Does GTK4 still have show_all for *some* things?

Reply:
@zeebok It doesn't, but we still need it in GTK 3 to show things packed into the dialog content area. I think `dialog.run` contains a `show_all` so we need to temporarily add it here when replacing with `dialog.present`